### ISSUE TITLE
IDISP008 should not warn when static context

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#207]
```cs
    public class C
    {
        public static readonly C Empty = new C(new Disposable());

        public C(IDisposable p)
        {
            P = p;
        }

        public IDisposable P { get; }

**Comment by JohanLarsson** [comment]
Maybe, not sure what is best here.
